src/scene/scene.js

```diff
@@ -137,6 +138,17 @@ class Scene extends EventHandler {
      */
     lightmapFilterEnabled = false;
 
+    /**
+     * Specify the pixel format to use for lightmaps. Can be:
```

Comment:
Maybe add some info about why you'd modify this property?

Reply:
I think instead of exposing the format, we should just expose 'lightmapHDR' flag or similar. And under the hood, use the format which is available, or automatically fallback to LDR.

We also need to only pick the HDR if OES_texture_float_linear is available, otherwise float textures cannot be linearly filtered, and only point sampling is allowed.

Otherwise, the user would need to understand this, and also we'd need to make these public: extTextureHalfFloat, textureHalfFloatRenderable, extTextureHalfFloat, textureHalfFloatUpdatable

Alternatively, we could make device.lightmapHdrFormat which would return format, or undefined if no hdr format is viable of similar.

Reply:
I thought about adding a flag, but didn't want to restrict users who may prefer one format over the other (when both are available). Is there a reason not to make the texture extensions public? They are being used all over the show.

Reply:
Perhaps a device getter called 'lightmapHdrFormat' as you suggestion that takes a boolean indicating whether to prefer 'larger' format (a.k.a 32F) over 'smaller' (16F)?

Reply:
yep, I'd be happy with that. Or maybe even `lightmapFormat(precision);` where precision can be 8, 16 or 32?
in that case, I guess we could just expose `scene.lightmapPrecision` (or similar name) to be one of those?